package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "VaultPress",
-	"version": "2.1.0-alpha",
+	"version": "2.1",
```

I would suggest sticking to full semantic versioning in `package.json`:

```suggestion
	"version": "2.1.0",
```